rest/models/recruitment.py

```diff
@@ -64,3 +65,38 @@ def to_json(rec, candidates_num):
             'major_name': rec.major.name,
             'major_mode': str(rec.major.mode)
         }
+
+    @staticmethod
+    def get_cycles_summary(recruitments):
+        """
+        Return json with summary of recruitments
+        :param recruitments: list of recruitments
+        :return: summary dict
+        """
+        rec = recruitments[0]
+        candidates_number = 0
+        overall_qualified = 0
+        min_point_limit = None
```

Comment:
it is better to init with "0" instead of None

Reply:
in case when all recruitment in cycles are active, they don't have calculated point limit. So when I init with `0`or `-1` it would look like one of recruitment in cycles ended, and that's not true.